content-security-policy/blink-contrib-2/base-uri-allow.sub.html.sub.headers

```diff
@@ -0,0 +1,6 @@
+Expires: Mon, 26 Jul 1997 05:00:00 GMT
+Cache-Control: no-store, no-cache, must-revalidate
+Cache-Control: post-check=0, pre-check=0, false
+Pragma: no-cache
+Set-Cookie: base-uri-allow={{$id:uuid()}}; Path=/content-security-policy/blink-contrib-2
+Content-Security-Policy: base-uri http://w3c-test.org; script-src 'self' 'unsafe-inline' http://w3c-test.org; connect-src 'self'; report-uri /content-security-policy/support/report.py?op=put&reportID={{$id}}
```

Comment:
I'll give it a try with "www1" instead and see if I can make it work and
not fail lint.

On Thu, Jun 25, 2015 at 10:41 AM Brad Hill <hillbrad@gmail.com> wrote:

> I developed and debugged this (base-uri) test locally, and deliberately
> picked stable resources on w3c-test.org to re-base against, so I believe
> it does work.  (this test required a good deal of manual work)
>
> On Thu, Jun 25, 2015 at 7:19 AM Kristijan Burnik <notifications@github.com>
> wrote:
>
>> In
>> content-security-policy/blink-contrib-2/base-uri-allow.sub.html.sub.headers
>> <https://github.com/w3c/web-platform-tests/pull/1935#discussion_r33259255>
>> :
>>
>> > @@ -0,0 +1,6 @@
>> > +Expires: Mon, 26 Jul 1997 05:00:00 GMT
>> > +Cache-Control: no-store, no-cache, must-revalidate
>> > +Cache-Control: post-check=0, pre-check=0, false
>> > +Pragma: no-cache
>> > +Set-Cookie: base-uri-allow={{$id:uuid()}}; Path=/content-security-policy/blink-contrib-2
>> > +Content-Security-Policy: base-uri http://w3c-test.org; script-src 'self' 'unsafe-inline' http://w3c-test.org; connect-src 'self'; report-uri /content-security-policy/support/report.py?op=put&reportID={{$id}}
>>
>> Will the base-uri present an issue when ran locally?
>> E.g. over http://web-platform.test:8000 ?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/w3c/web-platform-tests/pull/1935/files#r33259255>.
>>
>